packages/data-stores/src/help-center/actions.ts

```diff
@@ -128,7 +129,18 @@ export const setShowHelpCenter = function* (
 	show: boolean,
 	allowPremiumSupport = false,
 	options: HelpCenterShowOptions = { hideBackButton: false, searchTerm: '' }
-) {
+): Generator< unknown, { type: 'HELP_CENTER_SET_SHOW'; show: boolean }, unknown > {
+	const { isMinimized }: State = ( yield select( STORE_KEY, 'getState' ) ) as State;
```

Comment:
Let's try to use `getIsMinimized` directly!

Reply:
We can't use `getIsMinimized` directly because this action function doesn't have access to the redux state.